resume/src/components/Contact.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const Contact = (props) => {
+
+    const { email, phone, website, profiles, location } = props.basics
```

Comment:
nice way to deconstruct the basics object

Reply:
Thank you!